transports/quic/Cargo.toml

```diff
@@ -25,6 +25,7 @@ rustls = { version = "0.21.7", default-features = false }
 thiserror = "1.0.48"
 tokio = { version = "1.32.0", default-features = false, features = ["net", "rt", "time"], optional = true }
 socket2 = "0.5.4"
+ring = "0.16.20"
```

Comment:
I'd prefer if we don't depend on `ring` for this.

There is https://docs.rs/hkdf/0.12.3/hkdf/struct.Hkdf.html which is a small & pure Rust implementation.

Reply:
Yeah I thought of it reading your comment, but `quinn` requires [`impl HmacKey`](https://docs.rs/quinn/0.10.2/quinn/crypto/trait.HmacKey.html) which is only impl'ed for [`ring::hmac::Key`](https://docs.rs/quinn/0.10.2/quinn/crypto/trait.HmacKey.html#impl-HmacKey-for-Key) which we cannot impl on `Hkdf` due to the orphan rule.

Reply:
Wow that is a bit odd! Okay 🤷‍♂️